src/components/ContentWrapper.jsx

```diff
@@ -1,51 +1,35 @@
 import React from 'react'
-import { DOCTYPE_CONTACTS } from '../helpers/doctypes'
-import ContactsListDataLoader from './ContactsList/ContactsListDataLoader'
+import { PropTypes } from 'prop-types'
+
+import { Content } from 'cozy-ui/transpiled/react/Layout'
+
 import Header from './Header'
 import Toolbar from './Toolbar'
-import { Content } from 'cozy-ui/transpiled/react/Layout'
-import { Query } from 'cozy-client'
+import ContactsList from './ContactsList/ContactsList.jsx'
+import SpinnerContact from './Components/Spinner'
+import fetchSortedContacts from './Hooks/fetchSortedContacts'
+
+const ContentWrapper = ({ hasServiceBeenLaunched }) => {
+  const [hasContactsToShow, contacts] = fetchSortedContacts(
+    hasServiceBeenLaunched
+  )
+
+  if (!hasContactsToShow) {
```

Comment:
I don't like this wording since, if we have 0 contact, I expect to have "noContactsToShow" but I don't want to have a spinner.

Reply:
What about `isFetchFinished` ?